host/xtest/xtest_benchmark_2000.c

```diff
@@ -78,3 +79,61 @@ ADBG_CASE_DEFINE(XTEST_TEE_BENCHMARK_2002, xtest_tee_benchmark_2002,
 		/* How to implement */ ""
 		);
 
+
+/* ----------------------------------------------------------------------- */
+/* -------------------------- AES Benchmarks ----------------------------- */
+/* ----------------------------------------------------------------------- */
+
+static void xtest_tee_benchmark_2011(ADBG_Case_t *c)
+{
+	UNUSED(c);
+
+	int mode = TA_AES_ECB;	/* AES mode */
+	int decrypt = 0; /* Encrypt */
+	int keysize = AES_128;
+	size_t size = 1024;	/* Buffer size */
```

Comment:
I'm not convinced. You can always add a comment if needed:
```
aes_perf_run_test(TA_AES_ECB, 128 /* keysize */, 0 /* decrypt */,
                  1024 /* bufsize */, ...);
```

Actually I don't like the prototype of `aes_perf_run_test` much (too many parameters and will have to change if we add/remove options). Passing a struct would have been better I suppose, and I should have commented on that in you SHA PR. Anyway, I think we're bikeshedding, and I'm perfectly fine leaving the code as it is.